transport-native-kqueue/src/main/c/netty_kqueue_bsdsocket.c

```diff
@@ -159,8 +159,16 @@ static jobject netty_kqueue_bsdsocket_getPeerCredentials(JNIEnv *env, jclass cla
         (*env)->SetIntArrayRegion(env, gids, 0, 1, (jint*) &credentials.cr_gid);
     }
 
-    // TODO: getting the PID may require reading/sending "ancillary data" via SCM_CREDENTIALS which is not desirable.
-    return (*env)->NewObject(env, peerCredentialsClass, peerCredentialsMethodId, 0, credentials.cr_uid, gids);
+    pid_t pid;
```

Comment:
shouldn't this be initialised as 0?

Reply:
(otherwise we are passing in an uninitialised variable instead of 0 like in the previous version)

Reply:
You are right... I added the #ifdef to support really old build platforms, and forgot to initialise. Will commit a fix.